pkg/tracing/client/factory.go

```diff
@@ -16,18 +16,20 @@ import (
 	"gopkg.in/yaml.v2"
 
 	"github.com/thanos-io/thanos/pkg/tracing/elasticapm"
+	"github.com/thanos-io/thanos/pkg/tracing/google_cloud"
 	"github.com/thanos-io/thanos/pkg/tracing/jaeger"
 	"github.com/thanos-io/thanos/pkg/tracing/lightstep"
-	"github.com/thanos-io/thanos/pkg/tracing/stackdriver"
+	"github.com/thanos-io/thanos/pkg/tracing/migration"
 )
 
 type TracingProvider string
 
 const (
-	STACKDRIVER TracingProvider = "STACKDRIVER"
-	JAEGER      TracingProvider = "JAEGER"
-	ELASTIC_APM TracingProvider = "ELASTIC_APM"
-	LIGHTSTEP   TracingProvider = "LIGHTSTEP"
+	STACKDRIVER  TracingProvider = "STACKDRIVER"
+	GOOGLE_CLOUD TracingProvider = "GOOGLE_CLOUD"
```

Comment:
golang const should be camelCase https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps

Reply:
Looks like we have capitalized const names in other places handling configs (e.g. also in cache factory code), but I cannot find any reason for this. I'll fix it :+1: